src/components/create/MultipleChoices.tsx

```diff
@@ -70,70 +102,148 @@ const MultipleChoices = ({
+                                edge="end"
+                                sx={{ p: 0 }}
+                              />
                             }
-                            edge="end"
-                            sx={{ p: 0 }}
                           />
-                        }
-                      />
-                    </InputAdornment>
+                        </InputAdornment>
+                      }
+                    />
+                  </FormControl>
+                </Grid>
+                <Grid item xs={1} sx={{ textAlign: 'center' }}>
+                  {
+                    <IconButton
+                      data-cy={buildMultipleChoiceDeleteAnswerButtonCy(index)}
+                      type="button"
+                      disabled={choices.size <= 2}
+                      onClick={onDelete(index)}
+                    >
+                      <CloseIcon />
+                    </IconButton>
                   }
-                />
-              </FormControl>
+                </Grid>
+              </Grid>
             </Grid>
-            <Grid item xs={1} sx={{ textAlign: 'center' }}>
-              {
-                <IconButton
-                  data-cy={buildMultipleChoiceDeleteAnswerButtonCy(index)}
-                  type="button"
-                  disabled={choices.size <= 2}
-                  onClick={onDelete(index)}
+            {explanationList.get(index) || explanation ? (
+              <Grid container direction="column" key={index}>
+                <Grid item sx={{ paddingTop: 1.5 }}>
+                  <Typography variant="subtitle2" mb={1}>
+                    {t(
+                      'Type here an explanation on the correctness of this answer'
+                    )}
+                  </Typography>
+                </Grid>
+                <Grid item>
+                  <Grid
+                    container
+                    direction="row"
+                    key={index}
+                    alignItems="center"
+                  >
+                    <Grid
+                      item
+                      xs={11}
+                      style={{ display: 'flex', alignItems: 'center' }}
+                    >
+                      <FormControl variant="outlined" fullWidth>
+                        <InputLabel>
+                          {t(`Explanation ${readableIndex}`)}
+                        </InputLabel>
+                        <OutlinedInput
+                          data-cy={buildMultipleChoiceAnswerExplanationCy(
+                            index
+                          )}
+                          type="text"
+                          label={`Explanation ${readableIndex}`}
+                          value={explanation}
+                          placeholder={`Enter Explanation ${readableIndex}`}
+                          onChange={(e) => handleExplanationChange(index, e)}
+                        />
+                      </FormControl>
```

Comment:
I think you can use your sentence as the label, instead of having a random `Explanation x`.
Also, you can have as placeholder `Enter an explanation`.

Reply:
Perfect. But adding the sentence 'Type here an explanation on the correctness of this answer' as the InputLabel sentence will make the label too big once the user selects the TextField and the label is at the top. Should we add this sentence as the placeholder instead?